sd-validation/jsonld.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const walkObject = require('./helpers/walk-object.js');
+
+// This list comes from the JSON-LD 1.1 spec: https://json-ld.org/spec/latest/json-ld/#syntax-tokens-and-keywords
```

Comment:
That link is the older Community Group draft.  I'd suggest pointing at and tracking the current 1.1 Working Group draft:
```suggestion
// This list comes from the JSON-LD 1.1 editors draft:
// https://w3c.github.io/json-ld-syntax/#syntax-tokens-and-keywords
```